controllers/shooter.py

```diff
@@ -19,6 +20,7 @@ class ShooterController:
     shooter: Shooter
     turret: Turret
     vision: Vision
+    chassis: Chassis
```

Comment:
Did you mean to change this file?

Reply:
Yes, there is a call to `self.chassis.get_heading()` that will throw without it.

Reply:
I see the problem. Bad merges. Fixed now.